build.sbt

```diff
@@ -128,7 +129,6 @@ lazy val docs = project
     commonSettings,
     libraryDependencies ++= Seq(
       "org.mockito"       % "mockito-core" % MockitoVersion          % Test,
-      "org.scalatestplus" %% "mockito-3-2" % ScalatestMockitoVersion % Test,
```

Comment:
I think we should keep this here: mockito is an optional dependency, and it's up to the user to add it if they want.

Reply:
Why? It is same as scalatest. I created a new project, and copied to it [UserServiceSpec.scala](https://github.com/playframework/scalatestplus-play/blob/master/docs/manual/working/scalaGuide/main/tests/code/UserServiceSpec.scala), and it fails on the import of MockitoSugar. Why having one of them imported and not the other? It will cause the user to go and look for the compatible version, while you already did it.

Reply:
We need to choose whether we are providing a complete solution or we don't. If we decided to upgrade ScalaTest to beyond 3.0.8, we have to bring the dependencies to make it actually work, and not let the user to try all versions until he finds a good one. @raboof

Reply:
Another question we need to ask ourself is, will we have any user, that will use this library, and won't use mockito-3-2 ? The answer is probably no, because otherwise he won't have `MockitoSugar`.

Reply:
If you look at for example https://www.playframework.com/documentation/2.8.x/ScalaTestingWithScalaTest#Unit-Testing-Controllers, that is a useful scalatestplus-play spec that doesn't use mockito, right? or am I reading that wrong?

Reply:
You are right about the example you provided. But take a look at [this one](https://www.playframework.com/documentation/2.8.x/ScalaTestingWithScalaTest#Mockito). It won't work as it is.

Reply:
The reason it works at the moment is that mockito-3-2 is added at the `docs `project. A user that will copy paste it into a new project will fail importing.

Reply:
Exactly, so I propose we add the documentation on how to add the scalatestplus-mockito dependency to your build in that section. As far as I can see it is only used in that section, right? So it is an optional tool you _can_ use with `scalatestplus-play`, but not one that is required for all use cases.